src/Microsoft.CodeAnalysis.Analyzers/UnitTests/FixAnalyzers/FixerWithFixAllAnalyzerTests.Fixer.cs

```diff
@@ -306,21 +306,21 @@ Imports Microsoft.CodeAnalysis.CodeFixes
 Imports Microsoft.CodeAnalysis.CodeActions
 
 Class C1
-	Inherits CodeFixProvider
-	Public Overrides ReadOnly Property FixableDiagnosticIds() As ImmutableArray(Of String)
-		Get
-			Throw New NotImplementedException()
-		End Get
-	End Property
-
-	Public Overrides Function RegisterCodeFixesAsync(context As CodeFixContext) As Task
-		' Regular cases.
-		Dim codeAction1_1 = {|RS1010:CodeAction.Create(""Title1_1"", Function(x) Task.FromResult(context.Document))|}
-		Return Nothing
-	End Function
+    Inherits CodeFixProvider
+    Public Overrides ReadOnly Property FixableDiagnosticIds() As ImmutableArray(Of String)
+        Get
+            Throw New NotImplementedException()
+        End Get
+    End Property
+
+    Public Overrides Function RegisterCodeFixesAsync(context As CodeFixContext) As Task
+        ' Regular cases.
+        Dim codeAction1_1 = {|RS1010:CodeAction.Create(""Title1_1"", Function(x) Task.FromResult(context.Document))|}
+        Return Nothing
+    End Function
 
     Public Overrides Function GetFixAllProvider() As FixAllProvider
-	    Return WellKnownFixAllProviders.BatchFixer
+        Return WellKnownFixAllProviders.BatchFixer
```

Comment:
⚠️ All of this indentation should not have changed. The indentation is located within string literals. Applies to multiple files.